extension/lib/updateProductsInCart.js

```diff
@@ -0,0 +1,27 @@
+const ShopgateCartPipeline = require('./shopgate/CartExtensionPipeline')
+
+/**
+ * @param {PipelineContext} context
+ * @param {UpdateProductsInput} input
+ * @param {UpdateProductsCallback} cb
+ * @returns {Promise<void>}
+ */
+module.exports = async (context, input, cb) => {
```

Comment:
Due to latest changes in the SG Connect platform, async/await is truly supported and this needs to be updated. Callback is no longer needed therefore:
* response should be returned
* error should be thrown

Reply:
I have updated the file so it no longer uses a callback.